client/src/components/DocumentPage/ShareDocumentModal.jsx

```diff
@@ -5,6 +5,23 @@ import "./ShareDocumentModal.css";
 export default function ShareDocumentModal({ closeModal, documentId }) {
   const [email, setEmail] = useState("");
 
+  const handleShareDocument = async (documentId, email) => {
+    try {
+      const message = await shareDocument(documentId, email);
+      console.log(message);
+      if (message.message === "Added collaborator") {
```

Comment:
having logic based on strings is dangerous, what if the string changes or the capitalization does not match, etc... its safer to use constants like ENUM or just an object w/ a list of valid keys.